LICENSE

```diff
@@ -1,17 +1,15 @@
-#######
-License
-#######
+check_po, python package to automate task on traduction file .po
+Copyright (C) 2016  aRkadeFR
```

Comment:
@Natim I revert to What The Fuck License ?
(I would be interested by some explanation between these two licenses, as I mentioned, I only know the GPLv3).